docs/blog/2018-2-7-jam-out-your-blog/index.md

```diff
@@ -0,0 +1,158 @@
+
+### Here are some Pros (at the time of writing):
+
+*   Clean UI
+*   Free Tier
+*   User Roles (admins, editors, content authors)
+*   Comprehensive tutorials
+*   Fully-customizable front end
+*   Article search and filters
+
+That seems like a pretty solid win for Contentful, right? Well, there are a few issues. 
+
+While, a fully-customizable front end may seem really flexible, _**there is NOTHING set up when you create a new project.**_ You will have to create your own content models for blog posts and use GraphQL to query for each blog post type you set up. 
+
+<div style="width:100%;height:0;padding-bottom:55%;position:relative;"><iframe src="https://giphy.com/embed/16e1J9ztaZ3na"  width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/bored-kanye-west-16e1J9ztaZ3na">via GIPHY</a></p>
+
+
+If you’re like me and have no previous experience with Contentful, expect a few headaches. Your first time working with the service will involve a great deal of trial and error to get content/posts into your Gatsby website. Couple this with being a GraphQL newbie (like me), and you will get frustrated..._**very quickly**_.
+
+All I wanted to do was streamline my process and here I was adding another service into the mix and creating more work for myself. No bueno.
+
+To spare you the details, I spent a few hours comparing my GraphQL queries to those used in the [Gatsby + Contentful example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-contentful) to find errors in my code. I eventually gave up and I thought that there had to be an easier way to manage content online. Luckily there is. It's called Prose.
+
+## Managing Content Via Github: Enter Prose.io
+
+Prose was originally created for Jekyll by the people at [Development Seed](https://developmentseed.org/).
+
+Here is their description:
+
+_“Prose provides a beautifully simple content authoring environment for CMS-free websites. It's a web-based interface for managing content on GitHub. Use it to create, edit, and delete files, and save your changes directly to GitHub. Host your website on GitHub Pages for free, or set up your own GitHub webhook server._
+
+_Prose has advanced support for Jekyll sites and markdown content. Prose detects markdown posts in Jekyll sites and provides syntax highlighting, a formatting toolbar, and draft previews in the site's full layout.”_
+
+Source: [Prose](http://prose.io/#about)
+
+So, a service that works on top of Github where I, or anyone with a Github account can quickly create and manage posts. Sick! 
+
+All you have to do is set up a repository for the website, share access to anyone on your team, log in via [Prose.io](http://prose.io/), select your repository, and you’re good to go. You can edit existing posts, preview posts, and commit any changes directly within the interface. 
```

Comment:
"All you have to do" sounds like an advertisement, please remove.

Reply:
No problem.